src/redux/books/booksSlice.js

```diff
@@ -1,19 +1,13 @@
 import { createSlice } from '@reduxjs/toolkit';
 
-const initialState = {
-  books: [],
-};
-
 const booksSlice = createSlice({
   name: 'books',
-  initialState,
+  initialState: [],
   reducers: {
     addBook: (state, action) => {
```

Comment:
- [OPTIONAL] I would suggest to you set `state = initialState` to indicate the default value of state just as you indicate here but is the `initialState` really empty? :thinking: 
  _Probably, this is where you should add your book objects rather than have it exist outside the store_